src/OrbitQt/CaptureOptionsDialog.h

```diff
@@ -16,6 +17,24 @@
 
 namespace orbit_qt {
 
+namespace internal {
```

Comment:
I thought we didn't want nested namespaces? Why do you need it? Is this header file supposed to be used by a different module?

Reply:
I was missing the fact that we don't use it. I thought it is internal detail. Removed the namespace, as the header is not exposed.